csv2wiki/__init__.py

```diff
@@ -603,31 +615,33 @@ def __init__(self, config, csv_input, null_as_value, msg_out, dry_run_out):
             # There are two kinds of lines:
             #
             #   1) New section indicator (starts with dots)
-            #   2) Column number (starts with a number)
+            #   2) Text line (starts with a pipe)
+            #   3) Comment line (starts with a pound sign)
+            #      - This is not handled explicitly here because
```

Comment:
This surprises me -- that configparser would assume that its native comment syntax still applies even within the extended (multiline) value of a field -- but hey, if you tested it and that's the behavior, then that's the behavior.

Reply:
It confused me for a solid 5-10 minutes as I looked for where you were doing the work already that you wanted me to do :)